x/adminmodule/types/message_submit_proposal.go

```diff
@@ -77,22 +67,9 @@ func (m *MsgSubmitProposal) ValidateBasic() error {
 	if m.Proposer == "" {
 		return sdkerrors.Wrap(sdkerrors.ErrInvalidAddress, m.Proposer)
 	}
-
```

Comment:
wyt about adding a simple validation here e.g. len(m.Messages) > 0, m.Messages are of sdk.Msg type, maybe something else?